docs/csharp/programming-guide/concepts/linq/programming-with-nodes.md

```diff
@@ -130,7 +130,7 @@ AnAttribute="abc"  IsNamespaceDeclaration:False
  [!INCLUDE[sqltecxlinq](~/includes/sqltecxlinq-md.md)] allows for child text nodes of an <xref:System.Xml.Linq.XDocument>, as long as the text nodes contain only white space. However, the XPath object model does not include white space as child nodes of a document, so when you iterate through the children of an <xref:System.Xml.Linq.XDocument> using the <xref:System.Xml.Linq.XContainer.Nodes%2A> axis, white space text nodes will be returned. However, when you iterate through the children of an <xref:System.Xml.Linq.XDocument> using the XPath axis methods, white space text nodes will not be returned.  
```

Comment:
there are two occurrences of "white space text nodes" --> "white-space text nodes"